assets/app/styles/sidebar.css

```
@@ -1,423 +1,425 @@
+	border: 1px solid var(--bs-border-color, #444444);
+	border-radius: 4px;
+	font-weight: 500;
 }
 
 .auth-button.outline:hover {
-  background-color: var(--bs-surface-tertiary, #333333);
-  border-color: #555555;
-  border: 1px solid var(--bs-border-color, #444444);
+	background-color: var(--bs-surface-tertiary, #333333);
+	border-color: #555555;
+	border: 1px solid var(--bs-border-color, #444444);
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)